openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml

```diff
@@ -0,0 +1,104 @@
+description: Maximum Monthly Shelter Deduction by State and Household Size
+
+#NOTE: household sizes above 6 use the rate for a 6 member household in all states.   
+standard_deduction:
+  contiguous_US:
+    1:
+      2021-01-01: 167 
+    2:
+      2021-01-01: 167
+    3:
+      2021-01-01: 167
+    4:
+      2021-01-01: 181
+    5:
+      2021-01-01: 212
+    6:
+      2021-01-01: 243
+  AK_urban:
```

Comment:
Could you add a comment that values are identical across the Alaska subgeographies?

Reply:
Per our meeting, combine to AK